HW04/maksim_manzhai/spec/mentor_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'mentor'
+require 'student'
+require 'homework'
+require 'notification'
+
+RSpec.describe Mentor do
+  let(:mentor) { described_class.new(name: 'Will', surname: 'Smith') }
+  let(:student) { Student.new(name: 'Chris', surname: 'Rock') }
+  let(:notification) { Notification.new(title: 'HW03', description: 'OOP in Ruby') }
+  let(:homework) { Homework.new(title: 'HW03', description: 'OOP in Ruby', student: student, mentor: mentor) }
+
+  context 'when mentor created' do
+    it 'returns name' do
+      expect(mentor.name).to eql 'Will'
+    end
+
+    it 'returns surname' do
+      expect(mentor.surname).to eql 'Smith'
+    end
+  end
+
+  describe '#add_homework' do
+    it 'returns an object of class Homework' do
+      homework = mentor.add_homework(title: 'HW03', description: 'OOP in Ruby', student: student)
+      expect(homework).to be_a(Homework)
+    end
+  end
+
+  describe 'mark_as_read!' do
+    let(:notification) { Notification.new(title: homework.title, description: 'Homework started') }
+
+    it 'mentor see notification as unreaded' do
```

Comment:
please, think about all `it blocks` in your classes and refactor them
for instance, `it 'makes all notifications read'`